packages/dds/tree/src/feature-libraries/sequence-change-family/sequenceChangeset.ts

```diff
@@ -3,6 +3,17 @@
  * Licensed under the MIT License.
  */
 
+import { ChangeEncoder, JsonCompatible } from "../../change-family";
 import { Transposed as T } from "../../changeset";
 
 export type SequenceChangeset = T.LocalChangeset;
+
+export class SequenceChangeEncoder extends ChangeEncoder<SequenceChangeset> {
```

Comment:
we can just export a single ChangeEncoder<SequenceChangeset> instance. No need to export the class.

This will make future possible changes (like refactoring ChangeEncoder into an interface and implementing it with simple objects) less breaking.